internal/verification/phaverify.go

```diff
@@ -94,8 +102,19 @@ func (v *Verifier) VerifyDiagnosisCertificate(ctx context.Context, authApp *aamo
 			return nil, err
 		}
 
+		if cacheVal == nil {
+			return nil, fmt.Errorf("issuer not found: %v", claims.Issuer)
+		}
+
 		ha := cacheVal.(*model.HealthAuthority)
 
+		// Advisory check the aud.
+		if claims.Audience != ha.Audience {
+			logger.Errorw("certifice audience mismatch - will be a failure in the next release", "claims.Aud", claims.Audience, "allowed", ha.Audience, "iss", ha.Issuer)
+			// Hard error will start in next release.
+			// return nil, fmt.Errorf("audience mismatch for issuer: %v", ha.Issuer)
```

Comment:
Should we guard this with a feature flag or at least a greppable `TODO`?

Reply:
added feature flag - will file issue for removal